rastervision_pytorch_learner/rastervision/pytorch_learner/semantic_segmentation_learner.py

```diff
@@ -143,10 +143,11 @@ def build_model(self) -> nn.Module:
                 Parallel(old_conv, new_conv),
                 # sum the parallel outputs
                 AddTensors())
+        elif input_channels < old_conv.in_channels:
+            model.backbone.conv1 = nn.Conv2d(
+                in_channels=input_channels, **old_conv_args)
```

Comment:
I would recommend retaining the pretraining by copying over the weights of the first `input_channels` channels of the filters in the `old_conv`.

```python3
new_conv.weight.data[:, :input_channels] = old_conv.weight.data[:, :input_channels]
```

See example here:
https://colab.research.google.com/drive/1MKP6M-TXDoeuxGOumeQhuJFxjyZ-RfI4?usp=sharing

Reply:
You are right, I forgot that raster-vision does not deferentially train different parts of the network.